BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs

```diff
@@ -8,11 +16,37 @@ namespace BackendApartmentReservation.Controllers
     [ApiController]
     public class ProfilesController : ControllerBase
     {
+        private readonly UserManager<DbEmployee> _userManager;
         [HttpGet]
         [Route("profiles")]
         public async Task<IEnumerable<string>> Get()
         {
             return await Task.FromResult(new[] { "userProfile1", "userProfile2" });
         }
+
+        [HttpPost]
+        [AllowAnonymous]
+        [Route("profiles")]
+        public async Task<IActionResult> Register([FromBody] RegisterRequest model)
+        {
+            if (!ModelState.IsValid) return BadRequest(ModelState);
+            if (RegistrationInputValidator.IsInputValid(model)) return BadRequest("400 Bad Request");
```

Comment:
Once again, error string. You should not write 400 Bad Request, since that will be visible from HTTP response by `BadRequest` object. What you should do is something like `return BadRequest(ErrorCodes.InvalidEntity)` where `InvalidEntity` is something like `generic.invalid_entity`